x-pack/plugins/security_solution/public/timelines/components/timeline/session_tab_content/use_session_view.tsx

```diff
@@ -60,11 +60,18 @@ const NavigationComponent: React.FC<NavigationProps> = ({
   graphEventId,
   activeTab,
 }) => {
+  const title = useMemo(() => {
+    if (timelineId === TimelineId.active) {
+      return activeTab === TimelineTabs.graph ? i18n.CLOSE_ANALYZER : i18n.CLOSE_SESSION;
+    } else {
+      return graphEventId ? i18n.CLOSE_ANALYZER : i18n.CLOSE_SESSION;
+    }
+  }, [activeTab, graphEventId, timelineId]);
   return (
     <EuiFlexGroup alignItems="center" gutterSize="none">
       <EuiFlexItem grow={false}>
         <EuiButtonEmpty iconType="cross" onClick={onCloseOverlay} size="xs">
-          {activeTab === TimelineTabs.graph ? i18n.CLOSE_ANALYZER : i18n.CLOSE_SESSION}
+          {title}
```

Comment:
I'd like to have a unit test to check the rendered title for some of this deps: activeTab, graphEventId, timelineId

Reply:
@YulNaumenko added https://github.com/elastic/kibana/pull/129547/commits/7d92979a8f2d234579496dbc91ba0d1129251cbd